StartBonusActivation/handler.ts

```diff
@@ -324,24 +379,38 @@ export function StartBonusActivationHandler(
       .chain(_ => checkBonusActivationIsRunning(client, fiscalCode))
       .chain(_ => getLatestValidDSU(eligibilityCheckModel, fiscalCode))
       .chain((dsu: Dsu) =>
-        acquireLockForUserFamily(bonusLeaseModel, dsu.familyMembers).map(
-          _ => dsu
-        )
+        acquireLockForUserFamily(
+          bonusLeaseModel,
+          dsu.familyMembers
+        ).map(familyUID => ({ familyUID, dsu }))
       )
-      .chain((dsu: Dsu) =>
-        createBonusActivation(bonusActivationModel, fiscalCode, dsu)
+      .chain(({ familyUID, dsu }) =>
+        createBonusActivation(bonusActivationModel, fiscalCode, familyUID, dsu)
+      )
+      .chain(bonusActivation =>
+        runStartBonusActivationOrchestrator(
+          client,
+          bonusActivation,
+          fiscalCode
+        ).map(_ => bonusActivation)
```

Comment:
Maybe you can use `chainFirst` to ignore the return value here and pass the previous one